packages/analytics-browser-test/test/web-attribution.test.ts

```diff
@@ -644,8 +665,8 @@ describe('Web attribution', () => {
               events: [
                 generateAttributionEvent(++eventId, url, initReferrer),
                 generateSessionStartEvent(++eventId),
-                generatePageViewEvent(++eventId, 1, url, initReferrer),
-                generatePageViewEvent(++eventId, 2, newURL),
+                generatePageViewEvent(++eventId, 1, url, initReferrer, { previousPageUrl: '' }),
```

Comment:
The one lingering concern is this test, where I expected the previousPageUrl to be the initial url (`https://www.example.com?utm_source=test_utm_source`), but sessionStorage (which the plugin uses to store previous pages) was empty.